client/core/core_test.go

```diff
@@ -9388,3 +9410,75 @@ func TestFiatConversions(t *testing.T) {
 		t.Fatal("Expected fiat conversion to be disabled, all rate source data has expired.")
 	}
 }
+
+func TestValidateAddress(t *testing.T) {
+	rig := newTestRig()
+	defer rig.shutdown()
+	tCore := rig.core
+
+	wallet, tWallet := newTWallet(tUTXOAssetA.ID)
+	tCore.wallets[tUTXOAssetA.ID] = wallet
+
+	tests := []struct {
+		name string
+		addr string
+	}{{
+		name: "valid address",
+		addr: "randomvalidaddress",
+	}, {
+		name: "invalid address",
+		addr: "",
+	}}
+	for _, test := range tests {
+		tWallet.validAddr = test.addr != ""
+		valid := tCore.ValidateAddress(test.addr, tUTXOAssetA.ID)
+		if test.addr != "" != valid {
+			t.Fatalf("Got wrong response for address validation, got %v expected %v", valid, test.addr != "")
+		}
+	}
+}
+
+func TestEstimateSendTxFee(t *testing.T) {
+	rig := newTestRig()
+	defer rig.shutdown()
+	tCore := rig.core
+
+	wallet, tWallet := newTWallet(tUTXOAssetA.ID)
+	tCore.wallets[tUTXOAssetA.ID] = wallet
+
+	tests := []struct {
+		name    string
+		estFee  uint64
+		wantErr bool
+	}{{
+		name:    "want error",
+		estFee:  1e8,
+		wantErr: true,
+	}, {
+		name: "want zero fee value",
+	}, {
+		name:   "ok",
+		estFee: 1e8,
+	}}
+
```

Comment:
Should also add coverage for 

1. amount = 0
2. wallet not found
3. subtract = true and not withdrawer